web/app/templates/base.html

```diff
@@ -30,6 +31,16 @@
   {{ moment.include_moment() }}
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
   <script type="text/javascript">
+  
     var $SCRIPT_ROOT = {{ url_for('main.index', _external = True)|tojson|safe }}
+    var csrftoken = $('meta[name=csrf-token]').attr('content')
```

Comment:
The same problem appears hear; simply deleting the global variable would still not prevent malicios JS from sending **valid** ajax-requests.

## Suggested solution:
Protect from XSS-attacks in the first place, since that's where the real threat is